coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,118 @@
+from coala_quickstart.Strings import COALA_BEAR_LOGO, WELCOME_MESSAGES
+
+
+def text_wrap(*args, delimiter=' ', limit=80):
```

Comment:
didn't I leave a comment here that there seems to be something builtin you could use instead of this? CC @SanketDG again